config/initializers/active_support/cache/redis_store.rb

```diff
@@ -0,0 +1,22 @@
+module ActiveSupport
+  module Cache
+    class RedisCacheStore
+      def increment(name, amount = 1, options = nil)
```

Comment:
I would add comments to explain where this comes from and the why for this monkey patch